External/FEXCore/Source/Interface/Core/CPUID.cpp

```diff
@@ -872,6 +872,12 @@ FEXCore::CPUID::FunctionResults CPUIDEmu::Function_8000_0000h(uint32_t Leaf) {
 
 // Extended processor and feature bits
 FEXCore::CPUID::FunctionResults CPUIDEmu::Function_8000_0001h(uint32_t Leaf) {
+
+#ifndef _WIN32
```

Comment:
Perhaps add a comment above this with the rationale

Reply:
Added a comment